src/Tools/IdeBenchmarks/SQLitePersistentStorageBenchmark.cs

```diff
@@ -52,14 +51,8 @@ public void GlobalSetup()
     </Project>
 </Workspace>");
 
-            // Ensure we always use the storage service, no matter what the size of the solution.
```

Comment:
💡 Since this is an SQLite benchmark, can you set the database option here as well?

Reply:
we could.... except the point of the benchmark is to always use the db... so it would be weird to disable the DB...

Reply:
I mean set it to SQLite

Reply:
gotcha :)

Reply:
done.